controller/get.js

```diff
@@ -42,4 +42,13 @@ exports.viewFile = async(req,res)=>{
     } catch (error) {
         res.send(error);
     }
+}
+
+exports.viewComments = async(req,res)=>{
+    try {
+        var result = await IssueModel.findById(req.params.id);
+        res.send(result.comment);
```

Comment:
This looks strange .. `result.comment` is singular but it returns an array, also function name also implies to mutliple comments.

Reply:
Yeah it should be called comments